### PR TITLE
Fixed handling of the status field and its workflow

### PR DESCRIPTION
**Pull request by fgiorgetti**
@michalxo Please have a look and we can discuss about it tomorrow.
I have done some validation and it seems to be working just fine.

### CODE REVIEW

**Comment by michalxo** [comment]
LGTM. These changes are basically what I did, but on top of your branch now :-)
I am going to reuse it.

File: fmfexporter/adapters/polarion/polarion_test_case.py [hunk 1]
```diff
@@ -213,7 +221,17 @@ def to_xml(self):
...
         tc.set('id', self.id)
-        # tc.set('status', self.status)
+
+        # workflow is:
+        # - proposed if description, level and importance defined
+        # - approved if proposed conditions met, plus:
+        #   - automation_script set (when automated)
+        #   - requirement with role 'verifies' populated
+        if self.description != "" and self.level != "" and self.importance != "":
+            self.status = "proposed"
```
Comment (michalxo) [inline fmfexporter/adapters/polarion/polarion_test_case.py:231]: Where is this code?? :O

**Comment by fgiorgetti** [comment]
Lines 225..234.

Em ter, 5 de nov de 2019 07:17, Michal T <notifications@github.com>
escreveu:

**Comment by michalxo** [comment]
Devel branch ☺️

**Comment by fgiorgetti** [comment]
Yes :D

**Pull request merged**